verify.go

```diff
@@ -104,7 +104,7 @@ func VerifyWithPolicyCertConstraints(commonName string, dnsNames []string, email
 
 // Verify verifies a set of attestations against a provided policy. The set of attestations that satisfy the policy will be returned
 // if verifiation is successful.
-func Verify(ctx context.Context, policyEnvelope dsse.Envelope, policyVerifiers []cryptoutil.Verifier, opts ...VerifyOption) (map[string][]source.VerifiedCollection, error) {
+func Verify(ctx context.Context, policyEnvelope dsse.Envelope, policyVerifiers []cryptoutil.Verifier, opts ...VerifyOption) (bool, map[string]policy.StepResult, error) {
```

Comment:
This is a breaking API change. We'll need to deprecate the original function signature.

Reply:
Good point.